src/olympia/reviewers/models.py

```diff
@@ -427,16 +427,26 @@ def send_notifications(sender=None, instance=None, signal=None, **kw):
     if not subscribers:
         return
 
+    listed_perms = [
+        amo.permissions.ADDONS_REVIEW,
+        amo.permissions.ADDONS_CONTENT_REVIEW,
+        amo.permissions.ADDONS_RECOMMENDED_REVIEW,
+        amo.permissions.STATIC_THEMES_REVIEW,
+        amo.permissions.REVIEWER_TOOLS_VIEW
+    ]
```

Comment:
how does this differ from `is_user_any_kind_of_reviewer` - is it just it doesn't include unlisted review permission?

Reply:
Correct.

Reply:
I did consider writing a new acl helper method, but it would have been a super generic one and so far we'd only need it for this particular use-case. I also can't re-use much of what's already there since I need to use methods that don't rely on a `request`.